packages/api/methods.ts

```diff
@@ -115,79 +143,108 @@ export function importTransactions(
   });
 }
 
-export function getTransactions(accountId, startDate, endDate) {
+export function getTransactions(
+  accountId: APIAccountEntity['id'],
+  startDate: string,
+  endDate: string,
+) {
   return send('api/transactions-get', { accountId, startDate, endDate });
 }
 
-export function updateTransaction(id, fields) {
+export function updateTransaction(
+  id: TransactionEntity['id'],
+  fields: Partial<TransactionEntity>,
+) {
   return send('api/transaction-update', { id, fields });
 }
 
-export function deleteTransaction(id) {
+export function deleteTransaction(id: TransactionEntity['id']) {
```

Comment:
I think we're missing an equivalent `APITransactionEntity`, should we add one now or do you prefer on another PR?